newIDE/app/src/ExportAndShare/ShareDialog/ExportLauncher.js

```diff
@@ -80,6 +80,12 @@ export default class ExportLauncher extends Component<Props, State> {
   buildsWatcher = new BuildsWatcher();
   launchWholeExport: (i18n: I18nType) => Promise<void>;
 
+  componentWillMount() {
+    // Fetch limits when the export launcher is opened, to ensure we display the
+    // latest limits.
+    this.props.authenticatedUser.onSubscriptionUpdated();
```

Comment:
I had a case where the call to limits gave me the old value after the build finished (probably some latency on the backend side) and I realised I had not way of getting the right value, apart from going to my profile to retrigger a refresh. (I think it's fine to make this call on every mount?)